locking/api.go

```diff
@@ -190,6 +190,61 @@ func (c *lockClient) Search(remote string, searchReq *lockSearchRequest) (*lockL
 	return locks, res, err
 }
 
+// lockVerifiableRequest encapsulates the request sent to the server when the
+// client would like a list of locks to verify a Git push.
+type lockVerifiableRequest struct {
+	// Cursor is an optional field used to tell the server which lock was
+	// seen last, if scanning through multiple pages of results.
+	//
+	// Servers must return a list of locks sorted in reverse chronological
+	// order, so the Cursor provides a consistent method of viewing all
+	// locks, even if more were created between two requests.
+	Cursor string `json:"cursor,omitempty"`
+	// Limit is the maximum number of locks to return in a single page.
+	Limit int `json:"limit,omitempty"`
+}
+
+// lockVerifiableList encapsulates a set of Locks to verify a Git push.
+type lockVerifiableList struct {
```

Comment:
I named it based on `lockList`. Most of the names are based on the existing names for the search api call :)